core/src/main/java/io/grpc/internal/MessageFramer.java

```diff
@@ -218,28 +218,26 @@ private int writeKnownLengthUncompressed(InputStream message, int messageLength)
               String.format("message too large %d > %d", messageLength , maxOutboundMessageSize))
           .asRuntimeException();
     }
-    ByteBuffer header = ByteBuffer.wrap(headerScratch);
```

Comment:
FWIW, this allocation should exist on the stack.

Reply:
> FWIW, this allocation should exist on the stack.

What do you mean by "this allocation"?

Reply:
ByteBuffer.wrap() creates a new object.